[DO NOT MERGE] Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/jasp/jasp-desktop/) for JASP/JASP desktop.

FransMeerhoff commented:
Testing weblate

JoKeyser commented:
@FransMeerhoff , I just saw there is a conflict with `JASP-Desktop/po/jasp_de.po`, most likely due to the merge of #4031. Weblate also shows alerts and suggestions about that, https://hosted.weblate.org/projects/jasp/jasp-desktop/#alerts.

If you find the time, could you maybe explain a bit about the planned/expected timeline when the translations would be merged, and how translators should best test/use it at the moment? E.g. is Weblate just one of multiple candidates you are evaluating (see https://github.com/jasp-stats/jasp-issues/issues/726)?

Edit: Funny, even though the conflict is factually resolved, it's still showing up here on GitHub...  
Edit2: Ah, now it's resolved.

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 98.7% (4477 of 4532 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/de/

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 76.2% (1506 of 1975 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/de/

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 77.1% (1523 of 1975 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/de/

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 98.8% (4478 of 4532 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/de/

Commit:
Translated using Weblate (German)

Currently translated at 77.1% (1523 of 1975 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/de/

JoKeyser commented:
@FransMeerhoff , just in case it's not on your radar: Weblate is currently locked, "The translation was automatically locked due to following alerts: Could not merge the repository."

Commit:
Merge branch 'stable' into weblate-jasp-jasp-desktop
